test/suites/dev-tanssi-relay/ethereum-token-transfers/test_transfer_native_token.ts

```diff
@@ -109,9 +110,7 @@ describeSuite({
         it({
             id: "E02",
             title: "receive native token from Ethereum",
-            test: async ({ skip }) => {
-                // Uncomment once we have automated proof generation
-                skip();
+            test: async () => {
```

Comment:
Quick question, so now this test will fail if the proof generation changes, or do we need to manually change the proof generation script? I guess it depends on the kind of failure?

Reply:
Sorry, I didn't get you.

Reply:
But, yes, if proof generation specification changes then we need to manually update the script or at least the lodestar packages it depends on.

Reply:
The first time this test broke because of a new xcm version changing the message, so I guess if it happens again we will need to change some of the hardcoded data

Reply:
I am going to resolve this thread right now, but if you have more queries on this let's take it offline.